doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -0,0 +1,148 @@
+[id='test-scenarios-comparison-legacy-new-ref']
+
+= Feature comparison of legacy and new test scenario designer
+
+{PRODUCT} currently supports both the new *Test Scenarios* designer and the former *Test Scenarios (Legacy)* designer.
+
+The default designer is the new test scenarios designer, which supports testing of both rules and DMN models and provides an enhanced overall user experience with test scenarios. If required, you can continue to use the legacy test scenarios designer, which supports rule-based test scenarios only.
+
+IMPORTANT: The legacy Test Scenarios tool was deprecated with {PRODUCT} 7.3.0. It will be removed in a future {PRODUCT} release. Use the new Test Scenarios designer instead.
+
+The following table highlights the main features of legacy and new test scenario designer which are supported in {PRODUCT} to help you decide the suitable test scenarios designer in your project.
+
+* `+` indicates that the feature is present in the test scenario designer.
+* `-` indicates that the feature is not present in the test scenario designer.
+
+.Main features of legacy and new test scenario designer
+[cols="40%,10%,10%,40%", options="header"]
+|===
+|Feature & highlights
+|New designer
+|Legacy designer
+|Documentation
+
+a|*Creating and running a test scenario*
+
+* You can create test scenarios in {CENTRAL} to test the functionality of business rule data before deployment.
+* A basic test scenario must have at least a related data objects, *GIVEN* facts, and *EXPECT* results.
+* You can run the tests to validate your business rules and data.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about creating rule and DMN-based test scenarios, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about running the test scenarios, see xref:test-designer-run-test-proc[].
+* For more information about creating and running test scenarios(legacy), see xref:test-scenarios-legacy-create-proc[].
+
+a|*Adding GIVEN facts in test scenarios*
+
+You can insert and verify the *GIVEN* facts for the test.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about adding GIVEN facts in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding GIVEN facts in test scenarios(legacy), see xref:test-scenarios-legacy-GIVEN-proc[].
+
+a|*Adding EXPECT results in test scenarios*
+
+* It represents the objects and their fields whose exact values are checked based on the provided information.
+* The *EXPECT* section defines the expected results based on the *GIVEN* input facts.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about adding EXPECT results in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding EXPECT results in test scenarios(legacy), see xref:test-scenarios-legacy-EXPECT-proc[].
+
+|*KIE session*
+
+KIE session can be set on test scenario level settings.
```

Comment:
I'd suggest adding this sentence as a bullet point as well to follow the consistency.

Suggested rewrite:
You can set KIE session on test scenario level settings.